clusterloader2/testing/load/modules/reconcile-objects.yaml

```diff
@@ -0,0 +1,284 @@
+## Input params
+# Valid actions: "create", "scale and update", "delete"
+{{$actionName := printf "%s objects" .actionName}}
+{{$namespaces := .namespaces}}
+{{$tuning_set := .tuning_set}}
+
+# Derivative variables
+{{$is_creating := (eq .actionName "create")}}
+{{$is_scaling := (eq .actionName "scale and update")}}
+{{$is_deleting := (eq .actionName "delete")}}
+{{$randomScaleFactor := DefaultParam .randomScaleFactor 0}}
+{{$minReplicaFactor := SubtractFloat 1 $randomScaleFactor}}
+{{$maxReplicaFactor := AddFloat 1 $randomScaleFactor}}
+
+# DaemonSets
+{{$daemonSetImage := DefaultParam .daemonSetImage "k8s.gcr.io/pause:3.0"}}
+{{$daemonSetReplicas := .daemonSetReplicas}}
+
+# Deployments
+{{$bigDeploymentSize := .bigDeploymentSize}}
+{{$bigDeploymentsPerNamespace := .bigDeploymentsPerNamespace}}
+{{$mediumDeploymentSize := .mediumDeploymentSize}}
+{{$mediumDeploymentsPerNamespace := .mediumDeploymentsPerNamespace}}
+{{$smallDeploymentSize := .smallDeploymentSize}}
+{{$smallDeploymentsPerNamespace := .smallDeploymentsPerNamespace}}
+
+# StatefulSets
+{{$smallStatefulSetSize := .smallStatefulSetSize}}
+{{$smallStatefulSetsPerNamespace := .smallStatefulSetsPerNamespace}}
+{{$mediumStatefulSetSize := .mediumStatefulSetSize}}
+{{$mediumStatefulSetsPerNamespace := .mediumStatefulSetsPerNamespace}}
+
+# Jobs
+{{$bigJobSize := .bigJobSize}}
+{{$bigJobsPerNamespace := .bigJobsPerNamespace}}
+{{$mediumJobSize := .mediumJobSize}}
+{{$mediumJobsPerNamespace := .mediumJobsPerNamespace}}
+{{$smallJobSize := .smallJobSize}}
+{{$smallJobsPerNamespace := .smallJobsPerNamespace}}
+
+# PV
+{{$pvSmallStatefulSetSize := DefaultParam .pvSmallStatefulSetSize 0}}
+{{$pvMediumStatefulSetSize := DefaultParam .pvMediumStatefulSetSize 0}}
+
+## CL2 params
+{{$CHECK_IF_PODS_ARE_UPDATED := DefaultParam .CL2_CHECK_IF_PODS_ARE_UPDATED true}}
+{{$DISABLE_DAEMONSETS := DefaultParam .CL2_DISABLE_DAEMONSETS false}}
+{{$ENABLE_DNSTESTS := DefaultParam .CL2_ENABLE_DNSTESTS false}}
+{{$ENABLE_NETWORKPOLICIES := DefaultParam .CL2_ENABLE_NETWORKPOLICIES false}}
+{{$ENABLE_PVS := DefaultParam .CL2_ENABLE_PVS true}}
+
+steps:
+{{if $is_creating}}
```

Comment:
Please remove if. It is ok to start WaitForControlledPodsRunning multiple times (it will dedupe attempts to start)

Reply:
Preferably we can simply close all those measurements on the bottom of the file, but it's not possible for now.

Reply:
Ack.